hack/install-manifest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright 2021 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+for crd in config/crd/bases/*.yaml; do
+  kubectl apply -f "${crd}"
+done
```

Comment:
Kind of optional comment,
since there is only crds in `config/crd/bases/` dir  can't we just do `kubectl apply -f config/crd/bases/` instead of looping? and maybe since only we are running a single command have it in only Makefile with a target and drop this shell script?

Reply:
Hey! I'd rather loop through the YAML files since I can see each file being applied in the cluster. This is not a workflow that I'd like to add to `Makefile`. There's a lot in there, and I'd rather keep it simpler. It's only a shortcut for local tests only.

Reply:
Okay :)